### DISCUSSION TITLE
vue js

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20632] [Actions]
Как я могу найти документацию по vue js?)

**Comment by devkennyy** [comment]
Это ссылка на текущую версию Vue.js. [vuejs.org](https://vuejs.org/)

**Comment by mdnurulmomen** [comment]
Here you go [https://vuejs.org/](https://vuejs.org/)

**Comment by Akash1134** [comment]
Hi @MargaretRixten , thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
